internal/database/online/tikv/get.go

```diff
@@ -0,0 +1,98 @@
+package tikv
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/oom-ai/oomstore/internal/database/dbutil"
+	"github.com/oom-ai/oomstore/internal/database/online"
+	"github.com/oom-ai/oomstore/internal/database/online/redis"
+	"github.com/oom-ai/oomstore/pkg/oomstore/types"
+)
+
+type input struct {
+	entityKey string
+	feature   types.Feature
+}
+
+func (db *DB) Get(ctx context.Context, opt online.GetOpt) (dbutil.RowMap, error) {
+	// Proxy to MultiGet
+	res, err := db.MultiGet(ctx, online.MultiGetOpt{
+		Entity:      opt.Entity,
+		RevisionID:  opt.RevisionID,
+		EntityKeys:  []string{opt.EntityKey},
+		FeatureList: opt.FeatureList,
+	})
+	if err != nil {
+		return nil, err
+	}
+	if rowMap, ok := res[opt.EntityKey]; !ok {
+		return nil, fmt.Errorf("entity key %s not found", opt.EntityKey)
+	} else {
+		return rowMap, nil
+	}
+}
+
+func (db *DB) MultiGet(ctx context.Context, opt online.MultiGetOpt) (map[string]dbutil.RowMap, error) {
+	serializedRevisionID, err := redis.SerializeByValue(opt.RevisionID)
```

Comment:
How about moving `redis.SerializeByValue` to a util package? We probably use these functions in most kv stores.

Reply:
about to, though in separate PR